_posts/en/newsletters/2024-07-26-newsletter.md

```diff
@@ -0,0 +1,274 @@
+relay and fee bumping upgrades in Bitcoin Core.  Also included are our
+regular sections sharing popular questions and answers from the Bitcoin
+Stack Exchange, announcing new releases and release candidates, and
+describing notable changes to popular Bitcoin infrastructure projects.
+
+## News
+
+- **Varied discussion of free relay and fee bumping upgrades:** Peter
+  Todd [posted][todd fr-rbf] to the Bitcoin-Dev mailing list a summary
+  of a free relay attack that he previously [responsibly
+  disclosed][topic responsible disclosures] to Bitcoin Core developers.
+  This led to an entangled discussion of multiple issues and proposed
+  improvements.  Some of the topics discussed included:
+
+  - *Free relay attacks:* [free relay][topic free relay] occurs when a
+    full node relays unconfirmed transactions without the amount of
+    fee revenue in its mempool increasing by at least
+    the minimum relay rate (by default, 1 sat/vbyte).  Free relay often
+    costs some money, so it's not technically free, but the
+    cost is far below what honest users pay for relay.
+
+    Free relay allows an attacker to greatly increase the
+    bandwidth used by a relaying full nodes, which may reduce the number
+    of relay nodes.  If the number of independently operated relay nodes
+    drops too low, spenders are essentially sending transactions
+    directly to miners, which has the same centralization risks as
+    [out-of-band fees][topic out-of-band fees].
+
+    Todd's described attack exploits differences in mempool policy
+    between miners and users.  Many miners appear to enable
+    [full-RBF][topic rbf] but Bitcoin Core does not enable it by default
+    (see [Newsletter #263][news263 full-rbf]).  This allows an attacker
+    to craft different versions of a transaction which will be treated
+    differently by full-RBF miners and non-full-RBF relay nodes.  Relay
+    nodes can end up relaying multiple versions of a transaction that
+    have minimal chance of confirming, wasting the bandwidth of the
+    relaying nodes.
+
+    Free relay attacks do not directly allow attackers to steal other
+    users' funds, although a sudden or prolonged attack may be used to
+    disrupt the network and make other types of attacks easier.  To the
+    best of our knowledge, no disruptive free relay attack has ever been
+    performed.
+
+  - *Free relay and replace-by-feerate:* Peter Todd has previously
+    proposed two forms of replace-by-feerate (RBFR); see [Newsletter
+    #288][news288 rbfr].  One criticism of RBFR was that it enabled free
+    relay.  A similar amount of free relay is already possible through
+    the attack he described this week and similar attacks, so he
+    argued that concerns about free relay should not block adding a
+    feature useful for mitigating [transaction pinning attacks][topic
+    transaction pinning].
+
+    At least one [reply][harding rbfr fundamental] argued that free
```

Comment:
(I'm not sure about this suggestion; are there other replies that argue this? Seems like we should know if it was just one reply or more than one.)
```suggestion
    One [reply][harding rbfr fundamental] argued that free
```